### PR TITLE
337 - Adds AcceptField

### PR DESCRIPTION
**Pull request by acidjunk**
add acceptfield:

Screenshot 2023-11-16 at 17 09 15

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptFieldStyling.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+
+// import { DANGER, SUCCESS } from "../../../stylesheets/emotion/colors";
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptFieldStyling.ts:3]: This was my bad before transferring this story it to you: the proper colors from the theme must still be applied in this file.

Comment (acidjunk) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptFieldStyling.ts]: Fixed

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    className = '',
+    name,
+    onChange,
+    error,
+    showInlineError,
+    errorMessage,
+    data,
+    ...props
+}: AcceptFieldProps) {
+    const t = useTranslations('pydanticForms.fields');
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx:73]: Translations for other fields are located under the pydanticForms.widgets field

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    const { theme } = useOrchestratorTheme();
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx:74]: So you could use the useWithOrchestrator theme here to safe aline of code

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx (hunk 2 above)
Comment (acidjunk) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx:73]: Will this also be the placeholder/key for server sided translations?

Comment (acidjunk) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx:73]: Changed it :)

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx (hunk 3 above)
Comment (acidjunk) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx:74]: ✅

**Pull request merged**